Resources/Portals/_default/Default Website.template.nl-NL.resx

```diff
@@ -108,17 +108,18 @@
   <data name="Tab.ActivityFeed.FooterRightOuterPane.Module.Title.Text">
     <value xml:space="preserve">Verbind</value>
   </data>
-  <data name="Tab.ActivityFeed.FooterRightPane.Module.Content.0.Text">
-    <value xml:space="preserve">Awesome Cycles, Inc.</value>
+  <data name="Tab.ActivityFeed.FooterRightPane.Module.Content.0.Text" lastModified="2020-05-10 17:44:27">
+    <value xml:space="preserve">DNN Support B.V.</value>
```

Comment:
Who is DNN Support B.V. ?

Reply:
A demo name of a imaginary company. For Dutch you should have Dutch example, not something US/Inc.